docs/dox-ml-agents.conf

```diff
@@ -783,33 +783,10 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ML-Agents-Overview.md \
-                         Background-Unity.md \
-                         Background-Machine-Learning.md \
-                         Background-TensorFlow.md \
-                         Installation.md \
-                         Installation-Docker.md \
-                         Background-Jupyter.md \
-                         Getting-Started-with-Balance-Ball.md \
-                         Learning-Environment-Examples.md \
-                         Learning-Environment-Create-New.md \
-                         Learning-Environment-Design.md \
-                         Learning-Environment-Design-Agents.md \
-                         Learning-Environment-Design-Academy.md \
-                         Learning-Environment-Design-Brains.md \
-                         Training-ML-Agents.md \
-                         Training-PPO.md \
-                         Training-Curriculum-Learning.md \
-                         Training-Imitation-Learning.md \
-                         Using-TensorFlow-Sharp-in-Unity.md \
-                         Training-on-Amazon-Web-Service.md \
-                         Using-Tensorboard.md \
-                         Python-API.md \
-                         Feature-Broadcasting.md \
-                         Feature-Monitor.md \
-                         Limitations-and-Common-Issues.md \
-                         ../unity-environment/Assets/ML-Agents/Scripts \
-                         ../python/unityagents
```

Comment:
Why do we only include a portion of the files in the ../unity-environment/Assets/ML-Agents/Scripts and ignoring files like Communicator.cs and BCTeacherHelper.cs?

Reply:
We've only included the files that the developers will be using or extending. We do need to add ALL in the future.